lib/Runtime/Library/JavascriptFunction.h

```diff
@@ -42,6 +42,8 @@ namespace Js
         // Need a constructor cache on every function (script and native) to avoid extra checks on the fast path, if the function isn't fixed.
         Field(ConstructorCache*) constructorCache;
 
+        Field(bool) isJsBuiltInCode;
```

Comment:
>isJsBuiltInCode [](start = 20, length = 15)

I don't see this field is initialized anywhere? did you do that somewhere?

Reply:
This was inherited from the builtins branch. The constructor initialization was missing. It is set in the register* functions and used in a few places. Thanks for catching this!